draft-irtf-nwcrg-coding-and-congestion-03.xml

```diff
@@ -185,6 +202,10 @@
             which is useful input for a FEC coding scheme. Similarly, unequal error
             protection using fountain codes can be used to assign different priorities
 	    to blocks of data, and these priorities can be honored by a CC mechanism. --> </t> 
+
+    <t>The choice of the adequate transport layer may be related to application requirements:<list style="symbols"> 
+	<t>In the case of an unreliable live transfer, the transport layer may implement a non-reliable transport layer (e.g. UDP) or a partially reliable transport layer (e.g. DCCP <xref target="RFC8406"></xref> or those proposed by RMCAT WG). Depending on the amount of redundancy and network conditions, there could be cases where it ends up being impossible to carry traffic.</t>
+	<t>In the case of a reliable data transfer, the transport layer may implement a retransmission mechanism to guarantee the reliability of the file transfer (e.g. TCP). Depending on how the FEC and CC functions are scheduled (FEC above CC, FEC in CC, FEC below CC), the impact of a reliable transport on the FEC reliability mechanisms is different.</t></list></t>	
```

Comment:
This seems odd for several reasons:

In the case of an unreliable live transfer, the transport layer may implement a non-reliable transport layer (e.g. UDP) or a partially reliable transport layer (e.g. DCCP <xref target="RFC8406"></xref> or those proposed by RMCAT WG).

The RFC doesn't seem to be the right reference for DCCP; I think that neither DCCP nor the RMCAT mechanisms are partially reliable; I think a draft should not point at a WG. Also, the RMCAT stuff isn't an alternate protocol, it's just congestion control, over RTP/UDP, so it doesn't quite fit in the list. 
Also replacing "live" with "data" to be more generic, and fixing "layer may implement ... layer...", I'd suggest:

`In the case of an unreliable data transfer, the transport layer may employ an unreliable transport protocol (e.g., UDP or DCCP <xref target="RFC4340"></xref> or a partially reliable transport protocol (e.g., SCTP with partial reliabililty <xref target="RFC3758"></xref>).`

Reply:
That look fine for me. I meant unreliable with DCCP when I meant 'not in order'' - sorry ; should have been clearer. I have committed your suggestions.